frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -13,8 +13,6 @@ import { permissionCheck, PERMISSIONS } from "../../utils/PermissionsUtils";
 import { ReactComponent as RightLeftArrows } from "../../content/right-left-arrows.svg";
 import { useSessionContext } from "../../contexts/SessionContext";
 import { MemberType } from "../../hooks/UseGroups";
-import { getStoredOrg } from "../../contexts/SessionStorageTools";
```

Comment:
again, not sure why this file is in the diff. maybe this will sort itself out in a rebase of the base branch?